inst/config.yml

```diff
@@ -0,0 +1,372 @@
+
+asfd:
+  "ASFR MV":
+    func: get_array_data
+    rows: 2:8
+    type: "numeric"
+    dimensions: dimensions_agegr_year
+    convert_percent: TRUE
+
+sex_ratio_at_birth:
+  "SexBirthRatio MV":
+    func: get_tag_data
+
+births:
+  "Births MV":
+    func: get_tag_data
+
+total_net_migr:
+  "MigrRate MV2":
+    func: get_array_data
+    rows: c(3, 5)
+    type: "numeric"
+    dimensions: dimensions_sex_year
+  "MigrRate MV":
+    func: get_array_data
+    rows: c(4, 7)
+    type: "numeric"
+    dimensions: dimensions_sex_year
+
+net_migr_age_dist:
+  ## A row for each of the 17 age groups for male and female, 34 rows total
+  "MigrAgeDist MV2":
+    func: get_array_data
+    rows: 1 + 1:34
+    type: "numeric"
+    dimensions: dimensions_agegr_sex_year
+    convert_percent: TRUE
+  "MigrAgeDist MV":
+    func: get_array_data
+    rows: 5 + c(1:17 * 2, 37 + 1:17 * 2)
+    type: "numeric"
+    dimensions: dimensions_agegr_sex_year
+    convert_percent: TRUE
+
+hiv_pop:
+  "HIVBySingleAge MV2":
+    func: get_array_data
+    rows: 2:163
+    type: "numeric"
+    dimensions: dimensions_age_sex_year
+  "HIVBySingleAge MV":
+    func: get_array_data
+    rows: c(2:82, 84:164)
+    type: "numeric"
+    dimensions: dimensions_age_sex_year
+
+new_infections:
+  "NewInfectionsBySingleAge MV":
+    func: get_array_data
+    rows: c(0:80 * 3 + 2, 0:80 * 3 + 3)
+    type: "numeric"
+    dimensions: dimensions_age_sex_year
+
+art_pop:
+  "OnARTBySingleAge MV":
+    func: get_array_data
+    rows: c(0:80 * 3 + 2, 0:80 * 3 + 3)
+    type: "numeric"
+    dimensions: dimensions_age_sex_year
+
+adult_infec_reduc:
+  "AdultInfectReduc MV":
+    func: get_tag_data
+
+incid_pop_ages:
+  "EPPPopulationAges MV":
+    func: get_tag_data
+
+hiv_sex_ratio:
+  "HIVSexRatio MV":
+    func: get_tag_data
+
+dist_of_hiv:
+  "DistOfHIV MV2":
+    func: get_array_data
+    rows: 2:35
+    type: "numeric"
+    dimensions: dimensions_agegr_sex_year
+  "DistOfHIV MV":
+    func: get_array_data
+    rows: c(3:19, 21:37)
+    type: "numeric"
+    dimensions: dimensions_agegr_sex_year
+
+fertility_ratio:
+  "HIVTFR MV4":
+    func: get_array_data
+    rows: 1:7
+    type: "numeric"
+    dimensions: dimensions_agegr_year
+  "HIVTFR MV3":
+    func: get_array_data
+    rows: 1:7
+    type: "numeric"
+    dimensions: dimensions_agegr_year
+  "HIVTFR MV2":
+    func: get_array_data
+    rows: 1:6
+    type: "numeric"
+    ## this version of Spectrum stratified fertility reduction by
+    ## 15-17, 18-19, 20-24, ...
+    dimensions: dimensions_agegr_year_alternative
+  "HIVTFR MV":
+    func: get_array_data
+    rows: 1:7
+    type: "numeric"
+    dimensions: dimensions_agegr_year
+
+cd4_fertility_ratio:
+  "FertCD4Discount MV":
+    func: get_vector_data
+    cols: seq_len(DS) + 4
+    type: "numeric"
+  "fallback":
+    function(...) rep(1.0, DS)
+
+women_on_art:
+  "RatioWomenOnART MV2":
+    func: get_tag_data
+  "RatioWomenOnArt MV":
+    func: get_women_on_art
+  "fallback":
```

Comment:
multiline strings in yaml need special care.  If you look at
```
yaml::yaml.load_file("inst/config.yml")$women_on_art$fallback
```

you'll see you get lucky here because the line breaks get translated into a space.  Try instead:

```yaml
  "fallback": |-
    function(...) {
      stats::setNames(rep(1.0, fAG),
                      get_agegr_labels(between_15_49_only = TRUE))
    }
```

(and all other multiline strings)

Reply:
Thanks, fixed that now.